app/views/reminder_mailer/reminder_email.text.slim

```diff
@@ -0,0 +1,14 @@
+- rn2 = "\r\n\r\n"
```

Comment:
Hmm is this really necessary? Maybe adding empty lines in the template will have the same effect?